lib/evss/logged_service_exception.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module EVSS
+  class LoggedServiceException < EVSS::ServiceException
+    def initialize(res_body, user, req_body = nil)
+      @res_body = res_body
+      @user = user
+      @req_body = req_body
+
+      log_exception
```

Comment:
I think I'd prefer seeing `res_body, user, req_body` passed to log_exception as arguments rather than instance variables.  At first I thought they might be used elsewhere because they were instance variables.